service/interface.go

```diff
@@ -0,0 +1,15 @@
+package service
+
+import (
+	"github.com/ahab94/ApnaSchool/db"
+)
+
+// Service initializes our database instance
+type Service struct {
```

Comment:
This can be unexported right?

Reply:
If it becomes unexported, then i'll need a getter for it too to access in runtime.go